src/ndi-source.cpp

```diff
@@ -95,14 +104,41 @@ typedef struct {
 	bool audio_enabled;
 	ptz_t ptz;
 	NDIlib_tally_t tally;
+
+	ndi_source_config_t(int bandwidth_ = PROP_BW_HIGHEST,
+			    int latency_ = PROP_LATENCY_NORMAL)
+		: ndi_source_name(nullptr),
+		  ndi_receiver_name(nullptr),
+		  bandwidth(bandwidth_),
+		  behavior(BEHAVIOR_KEEP),
+		  remember_last_frame(true),
+		  sync_mode(PROP_SYNC_NDI_SOURCE_TIMECODE),
+		  framesync_enabled(false),
+		  hw_accel_enabled(false),
+		  yuv_range(video_range_type::VIDEO_RANGE_PARTIAL),
+		  yuv_colorspace(video_colorspace::VIDEO_CS_709),
+		  latency(latency_),
+		  audio_enabled(true),
+		  ptz(),
+		  tally()
+	{
+	}
```

Comment:
is this a typo ?

Reply:
No; that is intentionally an empty constructor.